UnitTests/DeckServiceTests.cs

```diff
@@ -12,45 +12,71 @@ namespace Backend.Tests
     public class DeckServiceTests
     {
         private readonly Mock<IDeckRepository> _mockDeckRepository;
+        private readonly Mock<IUserRepository> _mockUserRepository;
         private readonly DeckService _deckService;
 
         public DeckServiceTests()
         {
             _mockDeckRepository = new Mock<IDeckRepository>();
-            _deckService = new DeckService(_mockDeckRepository.Object);
+            _mockUserRepository = new Mock<IUserRepository>();
+            _deckService = new DeckService(_mockDeckRepository.Object, _mockUserRepository.Object);
         }
 
         [Fact]
         public void AddComment_ShouldAddCommentToDeck()
         {
             // Arrange
             var deckId = 1;
+            var username = "test_user";
             var commentDto = new CommentDTO
             {
                 Id = 1,
                 DeckId = deckId,
                 Text = "Great deck!",
+                Username = username,
                 CreatedAt = DateTime.UtcNow
             };
 
+            var user = new User
```

Comment:
Er der behov for at vi laver en user for at mappe den til DTO bagefter? kan vi ikke bare lave en DTO direkte?